workflow-service/src/main/java/com/redhat/parodos/project/service/ProjectServiceImpl.java

```diff
@@ -195,7 +195,7 @@ public AccessResponseDTO createAccessRequestToProject(UUID id, AccessRequestDTO
 		}
 		User user = userService.getUserEntityByUsername(accessRequestDTO.getUsername());
 		if (isNull(accessRequestDTO.getRole())) {
-			throw new ResourceNotFoundException("Role cannot be null");
+			accessRequestDTO.setRole(com.redhat.parodos.project.enums.Role.DEVELOPER);
```

Comment:
May I ask why this is the correct default role? 
Can you please document this behavior?

Reply:
Thanks :)